conformance/tests/httproute-timeout-backend-request.yaml

```diff
@@ -16,3 +16,12 @@ spec:
       port: 8080
     timeouts:
       backendRequest: 500ms
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /disabled-backend-timeout
```

Comment:
The test request is with `/disable-backend-timeout?delay=1s`, but path prefix here is `/disabled-backend-timeout`

There is discrepancy between test request path `/disable-backend-timeout?delay=1s` and path prefix in http route (e.g. `/disabled-backend-timeout`), which will cause 404 response instead of 200.